lib/riak/search/query.rb

```diff
@@ -0,0 +1,104 @@
+require 'riak/search'
+
+module Riak::Search
+  class Query
```

Comment:
+1. More documentation would have made it a bit easier for me to follow what is going on with the new classes.